package.json

```diff
@@ -29,22 +29,12 @@
   },
   "typings": "src/lib/index.d.ts",
   "scripts": {
-    "prettier": "prettier --single-quote --tab-width=2 --print-width=80 --semi --trailing-comma all",
-    "prettify": "yarn prettier --write 'src/**/*.{js,jsx,json,css,ts,tsx,html}'",
     "start": "set NODE_ENV=local && webpack-dev-server --config webpack/webpack.config.dev.js --content-base src/demo/dist --hot --inline --open --history-api-fallback",
-    "build:clean": "rimraf lib/",
     "build:lib": "webpack --config ./webpack/webpack.config.js && webpack --config ./webpack/webpack.config.min.js",
-    "build:gh-pages": "rimraf docs/ && . .env && mkdir -p docs && cp -a src/demo/dist/* docs/ && webpack --config ./webpack/webpack.config.gh-pages.js",
```

Comment:
necessary to have gh:pages build to generate gh:page

Reply:
I thought are not going to use rest of the packages scripts after my previous comment.
Will revert the change.

Reply:
Only GH page related and lib generation is good enough for me as a fair point.Agree to remove other unused packages

Reply:
It is now restored